src/hotspot/os/windows/os_windows.cpp

```diff
@@ -4235,11 +4235,11 @@ static void exit_process_or_thread(Ept what, int exit_code) {
   } else if (what == EPT_PROCESS) {
     ALLOW_C_FUNCTION(::exit, ::exit(exit_code);)
   } else { // EPT_PROCESS_DIE
-    ALLOW_C_FUNCTION(::_exit, ::_exit(exit_code);)
+    ::_Exit(exit_code);
```

Comment:
Ah, I was thinking the standard C method could be preferred over the Microsoft deprecated \_exit method, but I guess that's out of scope. I'm currently working on some build dev issue, I'll change this back later on

Reply:
Okay. The `_Exit` is the ISO C function, but POSIX also defines `_exit` (not that that is relevant to Windows). I can't see anything saying that Microsoft have deprecated `_exit` though:
https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/exit-exit-exit?view=msvc-170
If we were to change this I would still want the `ALLOW_C_FUNCTION`. Thanks

Reply:
I guess I can leave that \_Exit change for another day, it wouldn't make sense to have ALLOW\_C\_FUNCTION on something that is not banned by HotSpot after all

Reply:
For the record, `::_exit` _is_ forbidden.
https://github.com/openjdk/jdk/blob/7ac2f914aadf7fb12b7258e54efae959e15d6721/src/hotspot/share/utilities/globalDefinitions.hpp#L191
It might be deprecated as part of the optional defining and deprecation of POSIX names:
https://learn.microsoft.com/en-us/cpp/c-runtime-library/compatibility?view=msvc-170